docs/Running-the-server.md

```diff
@@ -18,12 +18,26 @@ There are currently two modes to run the Impostor server in. The first way is th
 ### Using docker
 
 [![Docker Image](https://img.shields.io/docker/v/aeonlucid/impostor?sort=semver)](https://hub.docker.com/repository/docker/aeonlucid/impostor)
-[![Docker Image](https://img.shields.io/docker/v/aeonlucid/impostor/edge)](https://hub.docker.com/repository/docker/aeonlucid/impostor)
 
 ```
 docker run -p 22023:22023/udp aeonlucid/impostor:nightly
 ```
 
+### Using docker-compose
+```
+version: '3.4'
+
+services:
+  impostor:
+    image: aeonlucid/impostor:nightly
+    container_name: impostor
+    ports:
+      - 22023:22023/udp
+    volumes:
+      - /path/to/local/config.json:/app/config.json # For easy editing of the config
+      - /path/to/local/plugins:/app/plugins # Only needed if using plugins
```

Comment:
Could we also add libraries, if that's the case?
`/path/to/local/libraries:/app/libraries  # Only needed if using external libraries`

Reply:
Sure!